scripts/python/tao_build.py

```diff
@@ -471,9 +471,6 @@ def bazel_build(target, flag=""):
             execute("sed -i \
                 's/values\[i\] = internal::InnerMostDimReducer<Self, Op>::reduce(\*this, firstIndex + i \* num_values_to_reduce,$/CoeffReturnType t = internal::InnerMostDimReducer<Self, Op>::reduce(\*this, firstIndex + i \* num_values_to_reduce, num_values_to_reduce, reducer)\;values\[i\] = t\;/g' \
                 'bazel-tf_community/external/eigen_archive/unsupported/Eigen/CXX11/src/Tensor/TensorReduction.h'")
-        if args.ral_cxx11_abi:
-            # A workaround for build compiler main with gcc 730 when enable cxx11 abi
-            execute("git am ../platform_alibaba/ci_build/patch/workaround-to-fix-cxx11-build.patch")
```

Comment:
Do we need to remove the patch file from the repo as well?

Reply:
The patch is not in this repo, this is what I want to fix at first, then I found it is not needed for cxx11 abi build, so just need to remove this logic from tao_build.py.